finalfrontier/src/skipgram_trainer.rs

```diff
@@ -186,6 +188,11 @@ where
                 (token * self.ctx_size * 2) + offset
             }
             ModelType::SkipGram => token,
+            ModelType::DirectionalSkipgram => {
+                let offset = if offset_idx < focus_idx { 0 } else { 1 };
+
+                (token * 2) + offset
```

Comment:
Only some cosmetic change:

```Rust
if offset_idx < focus_idx {
    token * 2
} else {
    token * 2 + 1
}
```

Reply:
I liked the original formulation ;)

Reply:
Though I was surprised that rustfmt would't split this in multiple lines.

Reply:
Well, I did approve the PR, so we can just ignore my comment ;)

Reply:
> Though I was surprised that rustfmt would't split this in multiple lines.

I've seen this in other places, too:

```Rust
        let epoch_status = if best_epoch == epoch { "🎉" } else { "" };
```

Taken from sticker!